### PR TITLE
Update readme with instruction to update version in rasax

### PR DESCRIPTION
**Pull request by sanchariGr**
**Proposed changes**:
- As per the last DoD in https://rasahq.atlassian.net/browse/ATO-929
- Updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -438,3 +438,11 @@ Copyright 2022 Rasa Technologies GmbH. [Copy of the license](LICENSE.txt).
...
 the bottom of the
 [Libraries Summary](https://libraries.io/github/RasaHQ/rasa).
+
+### Additional Release Tasks 
+In order to check compatibility between the new released Rasa version to the latest version of Rasax, we perform the following steps:
+1. Following a new Rasa release, an automated pull request is created in [Rasa-X-Demo](https://github.com/RasaHQ/rasa-x-demo/pulls). 
+2. Once the above PR is merged, follow instructions [here](https://github.com/RasaHQ/rasa-x-demo/blob/master/.github/VERSION_BUMPER_PR_COMMENT.md), to release a version.
+3. Update the new version in the Rasax [env file](https://github.com/RasaHQ/rasa-x/blob/main/.env).
+The [Rasa-X-Demo](https://github.com/RasaHQ/rasa-x-demo) project uses the new updated Rasa version to train and test a model which in turn is used by our CI to run tests in the Rasax repository, 
+thus validating compatibility between Rasa and Rasax.
```
Comment (ancalita) [inline README.md:448]:
```suggestion
### Additional Release Tasks 
In order to check compatibility between the new released Rasa version to the latest version of Rasa X/Enterprise, we perform the following steps:
1. Following a new Rasa release, an automated pull request is created in [Rasa-X-Demo](https://github.com/RasaHQ/rasa-x-demo/pulls). 
2. Once the above PR is merged, follow instructions [here](https://github.com/RasaHQ/rasa-x-demo/blob/master/.github/VERSION_BUMPER_PR_COMMENT.md), to release a version.
3. Update the new version in the Rasa X/Enterprise [env file](https://github.com/RasaHQ/rasa-x/blob/main/.env).
The [Rasa-X-Demo](https://github.com/RasaHQ/rasa-x-demo) project uses the new updated Rasa version to train and test a model which in turn is used by our CI to run tests in the Rasa X/Enterprise repository, 
thus validating compatibility between Rasa and Rasa X/Enterprise.
```

Corrected the usage of rasax with the one established in the docs.

Comment (ancalita) [reply README.md:448]: We need a clarification if we need these tasks for every micro? Or only for minor and major?

Comment (amalsgit) [reply README.md]: The automatic PR on the rasa-x-demo repo gets created for every OSS release. So I guess, it is probably up to us to decide if we want to follow these steps to bump the OSS version in `rasa-x` for every release or stick to minor and major releases.

Comment (sanchariGr) [reply README.md]: I've made a small edit to do this only when the latest version is the highest version.

**Pull request merged**